pages/learn/summer-school/quantum-simulation-summer-school-2022.vue

```diff
@@ -0,0 +1,161 @@
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/learnContent'
+import { Component } from 'vue-property-decorator'
+import CourseOverviewPage from '~/components/logic/CourseOverviewPage.vue'
+import { GeneralLink, StartLearningUrl } from '~/constants/appLinks'
+
+@Component({
+  head: () => ({
+    title: '2022 Qiskit global summer school on quantum simulation'
+  })
+})
+export default class SummerSchoolCoursePage extends CourseOverviewPage {
+  routeName: string = 'summer-school/quantum-simulation-summer-school-2022'
+
+  headerTitle = '2022 Qiskit global summer school on quantum simulation'
+  headerDescription = [
+    `This summer school provides a focused introduction to quantum computing
+    and its applications to quantum simulation, with a specific focus on
+    quantum chemistry. These lectures were first released as part of a two-week
+    intensive summer school in July 2022. This course contains 9 lectures
+    (split over 19 videos totalling ~17 hours) and 4 labs.`
+  ]
+
+  headerImg = '/images/learn/summer-school/quantum-simulation-summer-school-2022/header.png'
+
+  backToTextbookHomeLink: GeneralLink = {
+    url: '/learn',
+    label: 'Home',
+    segment: {
+      cta: 'back-to-textbook-home', location: 'header'
+    }
+  }
+
+  startLearningCTA: GeneralLink = {
+    url: StartLearningUrl.SummerSchool2022,
```

Comment:
the build is failing because this variable does not exist. it will need to be updated/added here: https://github.com/Qiskit/qiskit.org/blob/main/constants/appLinks.ts#L51-L58

Reply:
🤦 whoops sorry